src/queries.py

```diff
@@ -96,6 +96,6 @@ def with_params(
         name="Solver Slippage for Period",
         filepath="period_slippage.sql",
         v1_id=1728478,
-        v2_id=1836718,
+        v2_id=1956003,
```

Comment:
@fleupold - you will also want to do this in your PR (after changing the ownership of the query to the team account).

https://github.com/cowprotocol/solver-rewards/pull/188